eclair-core/src/main/scala/fr/acinq/eclair/io/OpenChannelInterceptor.scala

```diff
@@ -148,6 +148,14 @@ private class OpenChannelInterceptor(peer: ActorRef[Any],
 
   private def sanityCheckNonInitiator(request: OpenChannelNonInitiator): Behavior[Command] = {
     validateRemoteChannelType(request.temporaryChannelId, request.channelFlags, request.channelType_opt, request.localFeatures, request.remoteFeatures) match {
+      case Right(_: ChannelTypes.Standard) =>
+        context.log.warn("rejecting new obsolete incoming channels")
+        sendFailure("rejecting incoming channel: anchor outputs must be used for new channels", request)
+        waitForRequest()
+      case Right(_: ChannelTypes.StaticRemoteKey) if !nodeParams.channelConf.acceptIncomingStaticRemoteKeyChannels =>
+        context.log.warn("rejecting new static_remote_key incoming channels")
```

Comment:
Can you also update this error message, similarly to the one above, to tell peers to use anchor outputs?

Reply:
done in https://github.com/ACINQ/eclair/pull/2881/commits/401e3a89966c4e0d174959871252a82e9c7acbf7